src/eye_extractor/dr/hemorrhage_type.py

```diff
@@ -18,31 +18,31 @@ class HemorrhageType(enum.IntEnum):
 
 INTRARETINAL_PAT = re.compile(
     r'\b('
-    r'intraretinal\s*hem(orrhage|e)'
+    r'intraretinal\s*hem(orrhage|e)s?'
     r'|hem(orrhage|e)\s*intraretinal'
     r')\b'
 )
 DOT_BLOT_PAT = re.compile(
     r'\b('
-    r'dot blot\s*hem(orrhage|e)'
+    r'dot blot\s*hem(orrhage|e)s?'
     r'|hem(orrhage|e)\s*dot blot'
     r')\b'
 )
 PRERETINAL_PAT = re.compile(
     r'\b('
-    r'preretinal\s*hem(orrhage|e)'
+    r'preretinal\s*hem(orrhage|e)s?'
     r'|hem(orrhage|e)\s*preretinal'
     r')\b'
 )
 VITREOUS_PAT = re.compile(
     r'\b('
-    r'vitreous\s*hem(orrhage|e)'
+    r'vitreous\s*hem(orrhage|e)s?'
     r'|hem(orrhage|e)\s*vitreous'
     r')\b'
 )
 SUBRETINAL_PAT = re.compile(
     r'\b('
-    r'subretinal\s*hem(orrhage|e)'
+    r'subretinal\s*hem(orrhage|e)s?'

```

Comment:
All of these patterns have `hem(orrhage|e)s?` -- might consider refactoring that out into a variable so you only need to change it once.